_opcodes/op328-bgee.html

```diff
@@ -12,7 +12,7 @@
 pst: 0
 pstee: 1
 ---
-Sets the state of the targetted creature(s) to the value specified by the <code>Type</code> field.
+Sets the spell state of the targetted creature(s) to the value specified by the <code>Type</code> field.
```

Comment:
Spell and extended states are two different things.

Reply:
Opcode 328 can use both hardcoded states (IWD mode) and values from "SPLSTATE.IDS" (IWD2 mode) in BGEE, depending on the special parameter.

I could expand that phrase from `Sets the spell state...` to `Sets the extended or spell state..., depending on the "Mode" value.` to better account for the IWD mode. Thoughts?

Reply:
That and keeping the title simple and consistent. Or just reverting this file.

exstate.ids existed already in the originals btw.

Reply:
The reason for this change is [opcode 357](https://github.com/Gibberlings3/iesdp/pull/151/files#diff-2919fdce0f219fd67d06ecdb7acf306e662231d55246fb5d04f3c6592b64f94f) to avoid confusion.

I can change the title to `Set Extended or Spell State`, which would make it more consistent but not necessarily simpler. I'm open for suggestions.

Reply:
Ah, that's probably why the other is called "set status effects" in the original list. But I agree that state by itself evokes the bg1 states. The new title suggestion looks fine to me.